src/keyboard.rs

```diff
@@ -0,0 +1,1310 @@
+impl ModifiersState {
+    /// Returns `true` if the shift key is pressed.
+    pub fn shift_key(&self) -> bool {
+        self.intersects(Self::SHIFT)
+    }
+    /// Returns `true` if the control key is pressed.
+    pub fn control_key(&self) -> bool {
+        self.intersects(Self::CONTROL)
+    }
+    /// Returns `true` if the alt key is pressed.
+    pub fn alt_key(&self) -> bool {
+        self.intersects(Self::ALT)
+    }
+    /// Returns `true` if the super key is pressed.
+    pub fn super_key(&self) -> bool {
+        self.intersects(Self::SUPER)
+    }
```

Comment:
I would prefer them without the `_key` suffix as well. But `super` is a keyword in Rust so for the sake of consistency, this was the solution I chose here.

Reply:
Since this is not a widely used key, isn't it better to leave the others unchanged and make an exception for this one? That way 95%+ of uses aren't affected.

Reply:
> Since this is not a widely used key

The "Super" key is the Command key on macOS, which is as relevant to applications as the Control key on Windows/Linux. So I think it's warranted to name these consitstently.

Reply:
> The "Super" key is the Command key on macOS, which is as relevant to applications as the Control key on Windows/Linux.

Right. I had completely forgotten about that.

In any case, I'd consider this more of an unfortunate regression rather than a blocker (for now, at least).